test/negate.js

```diff
@@ -1,16 +1,16 @@
-var assert = require('assert');
-
 var R = require('..');
+var eq = require('./shared/eq');
 
 
 describe('negate', function() {
 
   it('negates its argument', function() {
-    assert.strictEqual(R.negate(-Infinity), Infinity);
-    assert.strictEqual(R.negate(-1), 1);
-    assert.strictEqual(R.negate(0), 0);
```

Comment:
The answer is actually `-0`, but `assert.strictEqual` doesn't differentiate `-0` from `0`.